src/directives/colony/incubate.ts

```diff
@@ -43,5 +46,18 @@ export class DirectiveIncubate extends Directive {
 				this.remove();
 			}
 		}
+
+		// if reserved or owned by Overmind user - throw warning but don't remove? Included code to remove, just commented out.
+		let AssimilatedRoomOwner = (typeof RoomIntel.roomOwnedBy(this.pos.roomName) === 'string' && RoomIntel.roomOwnedBy(this.pos.roomName) != MY_USERNAME && Assimilator.isAssimilated(RoomIntel.roomOwnedBy(this.pos.roomName)!))
```

Comment:
An example: 
```
let roomOwner = RoomIntel.roomOwnedBy(this.pos.roomName);
let NonConflictRoomOwner = (typeof roomOwner === 'string' && roomOwner != MY_USERNAME && (getAggressionLevel() == Aggression.LOW || Assimilator.isAssimilated(roomOwner!)));
```

Reply:
Similarly, it could be renamed from isAssimilated to isFriendly, which checks if you are set non-conflict and have not blacklisted them (enemy list), or if they are assimilated/on your whitelist (allies)